src/saturn_engine/worker_manager/config/declarative_job.py

```diff
@@ -96,7 +96,12 @@ def to_core_objects(
                     info=self.pipeline.to_core_object(),
                     args=dict(),
                 ),
-                labels=labels,
+                labels=dict(
+                    {
+                        "job-definition-name": name,
```

Comment:
should we not prefix it with `saturn/`?

Reply:
We could to avoid having people mistakenly override it in their topologies, but the metric service adds `saturn.job.labels` as a prefix so adding `saturn/` would result in the tag being : `saturn.job.labels.saturn/job-definition-name`.